<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalysisWindow</class>
 <widget class="QWidget" name="AnalysisWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnalysisWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
    outline: none;
    font: 14px &quot;微软雅黑&quot;;
    color: #484855;
    font-weight: bold;
    background-color: white;
}



QLineEdit{
    border: none;
    min-width: 170px;
    padding: 4px;
    border: 1px solid #484855;
}
QLineEdit:hover {
    min-width: 170px;
    border: 1px solid #57AEDA;
}


QTableWidget, QTextEdit{
    border: 1px solid #484855;
}
QTableWidget::item{
    selection-background-color: #d9e2e3;
    color: #484855;
}



QPushButton#btn_createGAT{
    border: 1px solid #60D16A;
    color: #60D16A;
}
QPushButton#btn_createGAT:hover{
    border: 1px solid #60D16A;
    background: #60D16A;
    color: white;
}
QPushButton#btn_createGAT:pressed {
    border: 1px solid #37BD42;
    background: #37BD42;
    color: white;
}


QPushButton{
    border: 1px solid #57AEDA;
    padding: 6px 24px;
    color: #57AEDA;
}
QPushButton:hover{
    border: 1px solid #57AEDA;
    background: #57AEDA;
    color: white;
}
QPushButton:pressed{
    border: 1px solid #2D94C7;
    background: #2D94C7;
    color: white;
}
QPushButton:disabled{
    border: 1px solid #aab;
    color: #aab;
}



QHeaderView::section {
    background: transparent;
    color:  #222230;
    padding-top: 6px;
    border: 1px solid #484855;
    border-left: none;
    border-top: none;
	padding-left: 8px;
	padding-right: 8px;
}



QScrollBar:vertical
{
    width:10px;
    background: white;
}
QScrollBar::handle:vertical
{
    width:8px;
    background: #ccd;
    border-radius: 3px;
    margin-left: 2px;
    margin-right: 2px;
}
QScrollBar::handle:vertical:hover
{
    background: #aab;
}
QScrollBar::add-line:vertical
{
    border-color: transparent;
}
QScrollBar::sub-line:vertical
{
    border-color: transparent;
}
QScrollBar::add-line:vertical:hover
{
    border-color: transparent;
}
QScrollBar::sub-line:vertical:hover
{
    border-color: transparent;
}



QScrollBar:horizontal
{
    height:12px;
    background: white;
}
QScrollBar::handle:horizontal
{
    height:8px;
    background: #ccd;
    border-radius: 4px;
    margin-top: 2px;
    margin-bottom: 2px;
}
QScrollBar::handle:horizontal:hover
{
    background: #aab;
}
QScrollBar::add-line:horizontal
{
    border-color: transparent;
}
QScrollBar::sub-line:horizontal
{
    border-color: transparent;
}
QScrollBar::add-line:horizontal:hover
{
    border-color: transparent;
}
QScrollBar::sub-line:horizontal:hover
{
    border-color: transparent;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="5,1">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="vl_input">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lb_inputString">
           <property name="text">
            <string>输入串：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_inputLL1"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tw_outputLL1">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_createGAT">
         <property name="text">
          <string>生成文法分析表</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hl_btns">
         <item>
          <widget class="QPushButton" name="btn_first">
           <property name="text">
            <string>FIRST集</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_follow">
           <property name="text">
            <string>FOLLOW集</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_firstvt">
           <property name="text">
            <string>FIRSTVT集</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_lastvt">
           <property name="text">
            <string>LASTVT集</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>文法分析表</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tw_analysisTable">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>算法优先关系表</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tw_operatorTable">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>原文法</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="te_showGrammar">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>消除左递归和二义性后的新文法</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="te_newGrammar">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
